compiler/src/dotty/tools/dotc/parsing/Parsers.scala

```diff
@@ -4103,10 +4177,10 @@ object Parsers {
       Template(constr, parents, derived, self, stats)
 
     def templateBody(parents: List[Tree], rewriteWithColon: Boolean = true): (ValDef, List[Tree]) =
-      val r = inDefScopeBraces(templateStatSeq(), rewriteWithColon)
+      val r = inDefScopeBraces(templateStatSeq(), rewriteWithColon = rewriteWithColon)
```

Comment:
We usually don't use named arguments when the rhs is the same. There's no point, since the rhs already tells us which argument we mean.